apps/src/util/loadingSkeleton.js

```diff
@@ -0,0 +1,13 @@
+// This is a _very_ thin convenience wrapper around react-loading-skeleton
+// to workaround an import issue and to also import the required css file.
+
+// Writing this as `import ... from 'react-loading-skeleton'` will cause
+// webpack to use index.mjs due to the way the package.json file for
```

Comment:
One more detail that I probably should have included is that our webpack config currently does not transpile .mjs files.  (I tried changing the webpack config to transpile .mjs but that caused other problems which I did not investigate further.)